scripts/setup.sh

```diff
@@ -340,6 +341,34 @@ else
   GCP_SCHEDULER_CRON="*/${QUERY_INTERVAL_MIN} * * * *"
 fi
 
+cd ../extensions || exit
+echo "- choosing extensions to upload to Dynatrace"
+for EXTENSION_ZIP in *.zip; do
+  EXTENSION_NAME="$(basename "$EXTENSION_ZIP" .zip)"
+  unzip -j -q "$EXTENSION_ZIP" "extension.zip"
+  unzip -p -q "extension.zip" "extension.yaml" > "$EXTENSION_NAME".yaml
+  EXTENSION_GCP_CONFIG=$(yq e '.gcp' "$EXTENSION_NAME".yaml)
+  SERVICES=$(echo "$EXTENSION_GCP_CONFIG" | yq e -j | jq -r 'to_entries[] | "\(.value.service)/\(.value.featureSet)"')
+  REMOVE_EXTENSION=true
+  for SERVICE in $SERVICES; do
```

Comment:
we also need to support case when service is configured without /fetureSet added, e.g. 'cloud_function' should be treated as a 'cloud_function/default'.
See https://github.com/dynatrace-oss/dynatrace-gcp-function/blob/master/src/main.py#L90